d-input.json

```diff
@@ -0,0 +1 @@
+[12,6,17,1,16,14,24,8,24,8]
```

Comment:
The generated input JSON file does not match our initial schema.
The initial schema should include the `number` key and it s array of objects.

But the current input file contains only an array of numbers. Please make the changes.